releasenotes/notes/37415.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+releaseNotes:
```

Comment:
Can you format the release note similar to this one?  There is a way to reference the GitHub Issue directly: https://github.com/istio/istio/blob/master/releasenotes/notes/24905.yaml

Reply:
```
issue:
- 37415
```